docs/patterns/components/LoaderOverlay/index.js

```diff
@@ -50,6 +45,15 @@ const LoaderOverlayDemo = () => {
         variant: {
           type: 'string',
           description: 'Loader variant, can be "small" or "large".'
+        },
+        focus: {
+          type: 'boolean',
+          description: 'whether or not to focus the loader on initial render'
+        },
```

Comment:
Updated the documentation/demo to reflect the new usage of the `focus` prop, which is similar to how the `Toast` component currently works.